### ISSUE TITLE
请问这个问题解决了吗?

### ISSUE THREAD

**Issue by jquanC** [DIYgod/APlayer#726]
中文用户请注意：请尽量用**英文**描述你的 issue，这样能够让尽可能多的人帮到你。

If you want to report a bug, please provide the following information:

-   The steps to reproduce.
-   A minimal demo of the problem via https://jsfiddle.net or http://codepen.io/pen if possible.
-   Which versions of APlayer, and which browser / OS are affected by this issue?

**Comment by jquanC** [comment]
Thanks! I will make it.




&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DIYgod/APlayer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月12日(星期二) 中午11:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DIYgod/APlayer] 请问‘lrc’歌词文件中的中文乱码问题解决了吗?Has the problem of garbled Chinese characters displayed in the ‘lrc’ lyrics file been resolved? (Issue #726)





 
用utf-8编码，别用gbk就行

**Comment by jack10082009** [comment]
In fact, you can use GBK encoding, but HTML pages are now declared as utf-8 encoding. If it is declared as GBK, it does work.
But,the UTF-8 is recommended.

**Comment by jquanC** [comment]
Thanks for the reply~&nbsp; I actually don't remember the details of using this plugin, although I use it on my blog and it's still online.But I remember&nbsp; utf8 supports Chinese encoding. The cause of this problem may be some codec inconsistencies.
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DIYgod/APlayer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月20日(星期四) 晚上9:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DIYgod/APlayer] 请问‘lrc’歌词文件中的中文乱码问题解决了吗?Has the problem of garbled Chinese characters displayed in the ‘lrc’ lyrics file been resolved? (Issue #726)





 
In fact, you can use GBK encoding, but HTML pages are now declared as utf-8 encoding. If it is declared as GBK, it does work.
 But,the UTF-8 is recommended.

**Comment by jack10082009** [comment]
Of course. utf-8 supports almost all characters, including Chinese characters of course. However, due to the different encoding mechanism, some GBK-encoded content is recognized as "undisplayable characters". If saved incorrectly, the famous "锟斤拷" will also appear.
